src/main.cpp

```diff
@@ -53,10 +53,12 @@ void setup() {
 void loop(){
   if(not alarm->isArmed()){
     if(alarm->isButtonPressed()){
+      SerialComm::sendLogMessage("Alarm Armed");
```

Comment:
So with how the Raspberry PI works, this line of code will significantly slows down the arming process because it keeps waiting for a response from the PI after sending this line of code. This should only run if Properties::Module_PI is set to true.